tests/constants.py

```diff
@@ -1,3 +1,25 @@
+import contextlib
+import os
+from collections.abc import Iterator
 from pathlib import Path
 
 REPO_ROOT = Path(__file__).parent.parent
+
+
+@contextlib.contextmanager
+def chdir(new_dir: Path) -> Iterator[None]:
```

Comment:
Moved this here, so I could use it without triggering the loading of the `CDFToml` before I could use this one.